AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Support/Internal/StorageMultipartUploadClient.swift

```diff
@@ -100,11 +100,7 @@ class DefaultStorageMultipartUploadClient: StorageMultipartUploadClient {
             var request = URLRequest(url: preSignedURL)
             request.cachePolicy = .reloadIgnoringLocalCacheData
             request.httpMethod = "PUT"
-
-            /*
-            let userAgent = AWSServiceConfiguration.baseUserAgent().appending(" MultiPart")
-            request.setValue(userAgent, forHTTPHeaderField: "User-Agent")
-             */
+            request.setValue(serviceProxy.userAgent, forHTTPHeaderField: "User-Agent")
```

Comment:
We no longer need the `"MultiPart"` suffix?

Reply:
We will need guidance on how metrics are tracked.